[2022-09-05 10:07:47,606] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: create_tables.test scheduled__2022-09-02T00:00:00+00:00 [queued]>
[2022-09-05 10:07:47,634] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: create_tables.test scheduled__2022-09-02T00:00:00+00:00 [queued]>
[2022-09-05 10:07:47,634] {taskinstance.py:1376} INFO - 
--------------------------------------------------------------------------------
[2022-09-05 10:07:47,635] {taskinstance.py:1377} INFO - Starting attempt 6 of 6
[2022-09-05 10:07:47,636] {taskinstance.py:1378} INFO - 
--------------------------------------------------------------------------------
[2022-09-05 10:07:47,703] {taskinstance.py:1397} INFO - Executing <Task(MySqlOperator): test> on 2022-09-02 00:00:00+00:00
[2022-09-05 10:07:47,710] {standard_task_runner.py:52} INFO - Started process 711 to run task
[2022-09-05 10:07:47,716] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'create_tables', 'test', 'scheduled__2022-09-02T00:00:00+00:00', '--job-id', '78', '--raw', '--subdir', 'DAGS_FOLDER/create.py', '--cfg-path', '/tmp/tmptly1poo8', '--error-file', '/tmp/tmpdr3j43z1']
[2022-09-05 10:07:47,717] {standard_task_runner.py:80} INFO - Job 78: Subtask test
[2022-09-05 10:07:47,917] {task_command.py:371} INFO - Running <TaskInstance: create_tables.test scheduled__2022-09-02T00:00:00+00:00 [running]> on host 613319c2eb0f
[2022-09-05 10:07:48,168] {taskinstance.py:1591} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=create_tables
AIRFLOW_CTX_TASK_ID=test
AIRFLOW_CTX_EXECUTION_DATE=2022-09-02T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=6
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-09-02T00:00:00+00:00
[2022-09-05 10:07:48,169] {mysql.py:82} INFO - Executing: SHOW VARIABLES LIKE "secure_file_priv";
[2022-09-05 10:07:48,207] {base.py:68} INFO - Using connection ID 'mysql_conn_id' for task execution.
[2022-09-05 10:07:48,321] {dbapi.py:231} INFO - Running statement: SHOW VARIABLES LIKE "secure_file_priv";, parameters: None
[2022-09-05 10:07:48,632] {dbapi.py:239} INFO - Rows affected: 1
[2022-09-05 10:07:48,884] {taskinstance.py:1420} INFO - Marking task as SUCCESS. dag_id=create_tables, task_id=test, execution_date=20220902T000000, start_date=20220905T100747, end_date=20220905T100748
[2022-09-05 10:07:49,130] {local_task_job.py:156} INFO - Task exited with return code 0
[2022-09-05 10:07:49,368] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
